### PR TITLE
Add note about hue argument unit in color augumentation.

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: dali/pipeline/operators/color/color_twist.cc [hunk 1]
```diff
@@ -55,7 +55,7 @@ DALI_SCHEMA(Hue)
     .NumInput(1)
     .NumOutput(1)
     .AddOptionalArg("hue",
-        R"code(Hue change in angles.)code", 0.f, true)
+        R"code(Hue change in angles [degrees].)code", 0.f, true)
```
Comment (mzient) [inline dali/pipeline/operators/color/color_twist.cc]:
```suggestion
        R"code(Hue change, in degrees].)code", 0.f, true)
```

File: dali/pipeline/operators/color/color_twist.cc [hunk 2]
```diff
@@ -76,7 +76,7 @@ DALI_SCHEMA(ColorTwist)
...
```
Comment (mzient) [inline dali/pipeline/operators/color/color_twist.cc]:
```suggestion
        R"code(Hue change, in degrees.)code", 0.f, true)
```

File: dali/pipeline/operators/color/color_twist.cc (hunk 1 above)
Comment (JanuszL) [reply dali/pipeline/operators/color/color_twist.cc]: Done

File: dali/pipeline/operators/color/color_twist.cc (hunk 2 above)
Comment (JanuszL) [reply dali/pipeline/operators/color/color_twist.cc]: Done

**Comment by JanuszL** [comment]
Builds 679722

**Pull request merged**